canopen/objectdictionary/__init__.py

```diff
@@ -219,6 +219,17 @@ def add_member(self, variable):
         self.subindices[variable.subindex] = variable
         self.names[variable.name] = variable
 
+class Unsigned24(struct.Struct):
+    def __init__(self, *args, **kwargs):
+        super(Unsigned24, self).__init__("<I", *args, **kwargs)
```

Comment:
Fixed the super thing.
This isn't an attempt to convert the uint24 to a regular 32-bit. The only reason I'm converting it to an ```I``` is because that's the closest data types ```Struct``` supports. 
Then I'm using ```Struct.unpack("<I)``` to make that 32-bit int to python int.

Reply:
I had to look it up in the Python library docs, but as I see it, the `<L` and `<I` formats are equivalent, both using four bytes for the conversion.  So your code seems correct, just wondering why not use the same letter when it doesn't really make a difference?